Cargo.toml

```diff
@@ -8,4 +8,4 @@ name = "osc"
 path = "src/osc-lib.rs"
 
 [dependencies]
-byteorder = "*"
\ No newline at end of file
+byteorder = "*"
```

Comment:
I'd recommend making this dependency a solid version (I don't think crates.io allows publishing crates that use the any-version specifier anymore due to the instability it can cause). The current stable version as of posting this comment is `0.4.2`.